app/config/config_test.yml

```diff
@@ -23,6 +23,10 @@ services:
     old_sound_rabbit_mq.referent_message_dispatcher_producer:
         class: 'Tests\AppBundle\Test\Producer\ReferentMessageNullProducer'
 
```

Comment:
Maybe add comment here to explain the reason of this service for the `test`  env.